applications/GeoMechanicsApplication/tests/cpp_tests/test_constitutive_law_set6.cpp

```diff
@@ -0,0 +1,56 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include "custom_utilities/constitutive_law_utilities.hpp"
+#include "includes/checks.h"
+#include "testing/testing.h"
+
+using namespace Kratos;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(SetSixConstitutiveParametersCorrectResults, KratosGeoMechanicsFastSuite)
+{
+    ConstitutiveLaw::Parameters ConstitutiveParameters;
+
+    Vector strain_vector(3);
+    strain_vector(0)           = 1.0;
+    strain_vector(1)           = 2.0;
+    strain_vector(2)           = 3.0;
```

Comment:
As for the other PR I have reviewed today, I would suggest to use the concise notation for assigning values to a vector (after including `boost/numeric/ublas/assignment.hpp`):
```suggestion
    strain_vector <<= 1.0, 2.0, 3.0;
```

Reply:
Thank you. fixed